### PR TITLE
resolution related hwobj

### PR DESCRIPTION
**Pull request by meguiraun**
small fixes

### CODE REVIEW

File: mxcube3/ho_mediators/beamline_setup.py [hunk 1]
```diff
@@ -679,7 +679,8 @@ def __init__(self, ho, name=''):
         super(ResolutionHOMediator, self).__init__(ho, name)
         ho.connect("valueChanged", self.value_change)
         self._precision = 3
-
+        self._ho.energy_hwobj = self.getObjectByRole("energy")
+        self._ho.dtox = self.getObjectByRole("dtox")
```
Comment (marcus-oscarsson) [inline mxcube3/ho_mediators/beamline_setup.py]: Hm, this should be done by the actual hardware object in this case Resolution, its at least the case in
HardwareObjects/Resolution.py. So there should not be need add this extra assignment.

Comment (marcus-oscarsson) [inline mxcube3/ho_mediators/beamline_setup.py]: You do need to have the correct configuration in the corresponding resolution.xml for that to work though

Comment (meguiraun) [inline mxcube3/ho_mediators/beamline_setup.py]: that's what I though! But then, the mockup does not do the job properly.  I just using master files...

Gonna check more carefully...

Comment (marcus-oscarsson) [inline mxcube3/ho_mediators/beamline_setup.py]: Ahh, yes Ive made some local changes to mockups that I did not commit yet. Ill commit those later

Comment (marcus-oscarsson) [inline mxcube3/ho_mediators/beamline_setup.py]: There are some a PR for the HardwareObjects that improve the ResolutionMockup object, give it a try.

**Comment by meguiraun** [comment]
with you hwobj pr this is no longer neeeded. Thanks

**Pull request closed without merging**